tests/integration/iop-integration-test-defaults.yaml

```diff
@@ -4,9 +4,9 @@ kind: IstioOperator
 spec:
   meshConfig:
     accessLogFile: "/dev/stdout"
-    #defaultConfig:
-    #  proxyMetadata:
-    #    ISTIO_META_DNS_CAPTURE: "ALL"
+    defaultConfig:
```

Comment:
other changes LGTM, is this expected though?

Reply:
yes, trying to get dns work in our tests on k8s. something is off - only vm works but not pods - esp for grpc